dbrp/service.go

```diff
@@ -265,7 +265,9 @@ func (s *Service) FindMany(ctx context.Context, filter influxdb.DBRPMappingFilte
 			if err != nil {
 				return false, ErrInternalService(err)
 			}
-
+			if defID == nil {
+				return true, nil
```

Comment:
I played around with this locally, and while it prevents the panic I think it results in too many results being returned because `filterFunc` is skipped. The biggest potential issue I see is skipping the filter on `OrgID`.